docs/dietpi_docs_todo.md

```diff
@@ -211,3 +213,26 @@ Check for this:
   - the warning *Documentation in progress…* should be deleted
   - the links on the top should be bold and link to the headings within the sub-page
 - In every sub-page the section *How do I run DietPi-Software and install Optimised software ?* should be present.
+
+### 10.3 Add software numbers to list?
+
+Q: Shall we add the software numbers to all optimized softwares?  
+Example: [29] TigerVNC Server - Desktop for remote connection
+
+Or shall this be done as a postfix instead of a prefix?
+
+#### Decision?
+
+## 11 Chapter for "hardening the system"?
+
+Add items about
+
+- Let's Encrypt: <https://github.com/MichaIng/DietPi/wiki/Let's-Encrypt---Certbot>
+- TLS-SSL: <https://github.com/MichaIng/DietPi/wiki/TLS-SSL>
+- Security recommendations: <https://github.com/MichaIng/DietPi/wiki/Security-recommendation>
+- Fail2Ban: <https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=30#p452>
+
+## 12 Folding@Home
+
+Q: Who knows, whether the issue ***Known limitation of using only little computational power*** is already present?  
+See <https://dietpi.com/phpbb/viewtopic.php?f=8&t=5&start=130#p13704> resp. [converted page](https://dietpi.com/docs/software/computational_science/).
```

Comment:
Hmm, I used the Windows client recently and it uses all 4 CPU threads as long as the work unit can do that. Yes, generally work units may only utilise a single core/thread, or at least temporarily for certain steps, but that should be only true in very rare cases where for some reason no multi-threading is possible. I don't remember seeing this on any recent work unit.

Adding multiple folding slots should generally decrease overall performance, so I'd not recommend this anymore but trust that the work unit "developers" creation those in a way to utilise the system capabilities in an optimal way and as well assigning the work units that can be processed fine by the own system. E.g. my notebook has only builtin graphics, hence the GPU can only handle a very limited kind of tasks, hence I usually get CPU-only work units.

Reply:
@MichaIng: Then I will kick this section out of the docu.